### PR TITLE
Allow to specify language id in command to create a new untitled file

### LINKED ISSUE #152920: Provide a command to create a new untitled file with a specific language set
VSCode provides `workbench.action.files.newUntitledFile` using which we can create a new untitled file, which then instructs to set a particular language:

newcreatepythonfile

We're working on our walkthrough in Python extension which directly offers to create a new python file, it would be great if the command supports an argument which accepts the language to create the file for.

cc/ @lramos15

Unfortunately we cannot use VSCode APIs to accomplish this as extension is not guaranteed to be activated when the walkthrough is opened, so the command isn't available when the button is clicked, see:

weirdcreatepythonfileexp

### PR DESCRIPTION
**Pull request by karrtikr**
This PR fixes https://github.com/microsoft/vscode/issues/152920

@bpasero @lramos15 Unfortunately this does not seem to be working even if I hardcode the language id as `"python"`, would be needing some help here.

### CODE REVIEW

**Comment by bpasero** [comment]
Looks good to me from editor service, i.e. when I run this code it opens a Python file:

```ts
editorService.openEditor({
	resource: undefined,
	options: {
		override: undefined,
		pinned: true
	},
	languageId: 'python',
});
```

**Comment by karrtikr** [comment]
Ah, actually for some reason the builder was not compiling my changes incrementally, which is why I got the impression it's not working. Does it not watch for changes by default?

I can confirm it works as well. I'll test out if walkthrough works as expected.

**Comment by lramos15** [comment]
> Does it not watch for changes by default?

It should but depends how you build. I just use `yarn watch` in my terminal and then using the "Launch VS Code" debug entry

**Comment by karrtikr** [comment]
Can confirm it works from welcome view 👍

> It should but depends how you build. I just use yarn watch in my terminal and then using the "Launch VS Code" debug entry

I see, I used the default build command `Ctrl+Shift+B`. The daemon watch process launched seems to exit automatically after a while.

File: src/vs/workbench/contrib/files/browser/fileCommands.ts [hunk 1]
```diff
@@ -629,28 +629,31 @@ KeybindingsRegistry.registerCommandAndKeybindingRule({
 			{
 				isOptional: true,
 				name: 'viewType',
```
Comment (lramos15) [inline src/vs/workbench/contrib/files/browser/fileCommands.ts:631]: Can we set this name to be something meaningful now that it's not just view type

**Review by lramos15 — CHANGES_REQUESTED** [review]
Just a small nit

**Review by lramos15 — APPROVED** [review]

**Pull request merged**